src/common/cohort_lru.h

```diff
@@ -139,15 +139,15 @@ namespace cohort {
 	for (int ix = 0; ix < n_lanes; ++ix,
 	       lane_ix = next_evict_lane()) {
 	  Lane& lane = qlane[lane_ix];
-	  lane.lock.lock();
+	  std::unique_lock<std::mutex> lane_lock{lane.lock};
```

Comment:
I'd prefer not specifying the exact type of `lane_lock`. as, the type of `Lane::LK` is template parameter of `LRU`. and C++17 is able to deduce the type of `lane_lock` even we don't specify it, so following code should compile and work:
```c++
std::unique_lock lane_lock{lane.lock};
```

Reply:
Well. pass complication and testcases

Reply:
recommit it